thesis-de/content-de.tex

```diff
@@ -1713,11 +1713,13 @@ \subsubsection{Durchführung Evaluierung Frontend}
 
 \paragraph{Gridster-D3}
 \label{paragraph:gridster-d3}
-Gidster ist eine von Anmol Koul entwickelte Softwarelösung, welche auf
-OpenSource Basis auf GitHub liegt. Während der Evaluierung von dieser Software,
-ist aufgefallen, dass es sich hierbei nicht direkt um eine
-Visualisieurngssoftware handelt, sondern lediglich eine Funktion innerhalb
-einer Weboberfläche, nämlich das Drag \& Drop, von einzelnen Graphen.
+Gridster ist eine von Anmol Koul entwickelte OpenSource Softwarelösung, welche
+auf \gls{GitHub} einsehbar und dokumentiert ist. Während der Evaluierung von
+dieser Software ist aufgefallen, dass es sich hierbei nicht direkt um eine
+Visualisierungssoftware handelt, sondern lediglich um eine Funktion, nämlich das
+Drag \& Drop, von einzelnen Graphen innerhalb einer Weboberfläche. Mit dieser
+hat man die Möglichkeit, ein Dashboard, welches selbst entwickelt worden ist
+individuell anpassen zu können, Graphen zu verschieben oder direkt zu löschen.
```

Comment:
wenn ich das richtig sehe geht es bei gridster um das verschieben von div containern auf einer html seite.

https://anmolkoul.wordpress.com/2015/07/06/drag-and-drop-visuals-in-your-interactive-dashboard-gridster-d3-js/

die graphen sind da nur zufällig drinn